app/ui/static/styles.css

```
@@ -14,13 +14,41 @@
   --success: #2ba471;
   --warning: #d8922a;
   --danger: #e24c4c;
+  --surface-soft: #f8f9fb;
+  --surface-muted: #f3f5f7;
+  --text-body: #2f3642;
+  --text-secondary: #3a404c;
+  --chart-grid: rgba(72, 82, 102, .12);
   --radius: 10px;
   --radius-sm: 6px;
   font-family: "PingFang SC", "Microsoft YaHei UI", "Microsoft YaHei", system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)